<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Gestor de archivos MyPime</string>
  </property>
  <widget class="QPushButton" name="BtnCrear">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Escribir en archivo</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>258</width>
     <height>213</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Leer</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="TxtLeer"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BtnLeer">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>LeerArchivo</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>258</width>
     <height>213</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Escribir</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="TxtEscribir"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BtnLeerXLinea">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>LeerXLinea</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>BtnCrear</tabstop>
  <tabstop>TxtEscribir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
